From: Josh Law <hlcj1234567@gmail.com>
Subject: lib: glob: add explicit include for export.h
Date: Sun, 1 Mar 2026 15:21:41 +0000

Include <linux/export.h> explicitly instead of relying on it being
implicitly included by <linux/module.h> for the EXPORT_SYMBOL macro.

Link: https://lkml.kernel.org/r/20260301152143.2572137-1-objecting@objecting.org
Signed-off-by: Josh Law <objecting@objecting.org>
Reviewed-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/glob.c |    1 +
 1 file changed, 1 insertion(+)

--- a/lib/glob.c~lib-glob-add-explicit-include-for-exporth
+++ a/lib/glob.c
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: (GPL-2.0 OR MIT)
 #include <linux/module.h>
 #include <linux/glob.h>
+#include <linux/export.h>
 
 /*
  * The only reason this code can be compiled as a module is because the
_
